VRCs/vrc-15.md

```diff
@@ -59,7 +59,9 @@ Application builders can define arbitrary compute jobs, that once audited and ap
 
 ## Permissions and Payment Contracts
 
-The Query Engine contract handles fine-grained permissions and defining pricing for data access. It allows a DLP to grant access to specific schemas, tables and columns to a specific user for a pre-negotiated price. In the beginning, payment will be in $VANA but eventually will be in DLP-specific tokens. 
+The Query Engine contract handles fine-grained permissions and defining pricing for data access. It allows a DLP to grant access to specific schemas, tables and columns to a specific user for a pre-negotiated price. 
+
+Payments will be in $VANA. 20% of the data access fees will be distributed back to TEE stakers as yield and to TEE operators to cover operational costs. This ratio may be adjusted over time. The remaining portion will go to the DLP treasury in v0. Starting in v1, it'll be used to buy and burn DLP tokens.
```

Comment:
```suggestion
Payments will be in $VANA. 20% of the data access fees will be distributed back to TEE stakers as yield and to TEE operators to cover operational costs. This ratio may be adjusted over time and may eventually involve a dynamic pricing mechanism. The remaining 80% will go to the DLP treasury in v0. Starting in v1, it'll be used to buy and burn DLP tokens. The 20% rate is deliberately set on the higher side initially, as it's preferable to start higher and reduce if needed, rather than starting too low and having to introduce unexpected increases later.
```